subscription.go

```diff
@@ -0,0 +1,133 @@
+package stripe
```

Comment:
I see the rational here, but I wonder if we should leave this as `sub.go` so that you can always match the name of a Go file with the name of the corresponding package? (`sub/`)

Reply:
Agreed, I just renamed those back.